app/helpers/application_helper.rb

```diff
@@ -23,7 +23,12 @@ def avatar(email = nil)
     else
       gravatar_id = Digest::MD5.hexdigest(email.downcase)
       gravatar_size = Rails.application.config.gravatar_size
-      tag :image, src: "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{gravatar_size}&d=#{CGI.escape(root_url + default)}"
+      if Rails.env.development? 
```

Comment:
@Alwahsh I think its better keep this as a local check rather than defining it globally.